src/lender/reserve.sol

```diff
@@ -130,6 +130,7 @@ contract Reserve is Math, Auth {
             if(drawAmount > left) {
                 drawAmount = left;
             }
+
```

Comment:
we need a check in the  _deposit function if the epoche is in submission before we call wipe, otherwise repayments will fail. As wipe is only possible if epoch is not in submission period

Reply:
Yes, still on my todo list. I will push it in a separated PR